src/test/java/com/coherent/zipcode/zipCodeController/ZipCodeGET.java

```diff
@@ -0,0 +1,41 @@
+package com.coherent.zipcode.zipCodeController;
+
+import com.coherent.HttpRequestManager;
+import com.coherent.ZipCodeController;
+import com.coherent.zipcode.BasicTestClass;
+import com.google.common.collect.Sets;
+import lombok.SneakyThrows;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+
+public class ZipCodeGET extends BasicTestClass {
+
+    @Test
+    public void shouldGetZipCodeWithGET() {
+        List<String> expectedZipCodes = new ArrayList<>(Arrays.asList("12345" ,"23456", "ABCDE"));
+        List<String> zipCodesResponse = zipCodeController.sendGetZipCodes(201);
+
+        org.assertj.core.api.Assertions.assertThat(zipCodesResponse).containsAll(expectedZipCodes);
+        //QUESTION: why checking size is a valid assertion here?
```

Comment:
Could be valid when you know default size of your array(which is your case). So you can control that state doesn't change by assertion. In the same time you have defined list of expected values, so containsAll check will be enough

Reply:
got it, thank you!